src/components/tl/ucp/tl_ucp.c

```diff
@@ -27,6 +29,16 @@ static ucs_config_field_t ucc_tl_ucp_context_config_table[] = {
      ucc_offsetof(ucc_tl_ucp_context_config_t, preconnect),
      UCC_CONFIG_TYPE_UINT},
 
+    {"NPOLLS", "10",
+     "Number of ucp progress polling cycles for p2p requests testing",
+     ucc_offsetof(ucc_tl_ucp_context_config_t, n_polls),
+     UCC_CONFIG_TYPE_UINT},
+
+    {"KN_BARRIER_RADIX", "4",
```

Comment:
I would suggest to follow some naming convention for parameters. For me it looks better something like
"COLLECTIVE_ALGORITHM_PARAMETER". For example BARRIER_KN_RADIX=4 and ALLTOALL_PAIRWISE_CHUNK=1 instead of KN_BARRIER_RADIX=4 and PAIRWISE_ALLTOALL_CHUNK=1

Reply:
👍